include/faabric/scheduler/FunctionCallClient.h

```diff
@@ -42,6 +45,9 @@ class FunctionCallClient : public faabric::transport::MessageEndpointClient
 
     faabric::HostResources getResources();
 
+    void sendAddPendingMigration(
+      std::shared_ptr<faabric::PendingMigrations> req);
```

Comment:
Is the `Add` necessary in here? I think `sendPendingMigration` conveys the meaning and is easier to read.

Reply:
Agree, also the use of the plural and singular form of migration(s) was confusing. I have renamed the function calls to `sendPendingMigrations` and `recvPendingMigrations`.